docs/blog/newsletter-february-2021.rst

```diff
@@ -0,0 +1,88 @@
+
+A middle-ground approach is using text descriptions of UI elements, like "Click the START button", as it's easier to keep text descriptions matching the UI. And well-designed user interfaces and UI microcopy often mean that users don't need screenshots to find their way through the product.
+
+What about a video walkthrough or GIFs? Again, documentarians were split: it's helpful to watch a process through before trying it yourself, but videos and GIFs are inaccessible for people with visual impairments, distracting for people with attention issues, and unusable for people who don't have dependable broadband service. And following video instructions can be frustrating if you have to repeatedly press pause, or feel like the pace of spoken instructions is too slow.
+
+For more on screenshots in documentation, check out Steve Stegelin's `Graphic Content Warning: The Pros, Cons, and Alternatives to Screenshots <https://www.writethedocs.org/videos/portland/2018/graphic-content-warning-the-pros-cons-and-alternatives-to-screenshots-steve-stegelin/>`_ from Write the Docs Portland 2018 and Swapnil Ogale's `When bad screenshots happen to good writers <https://www.writethedocs.org/videos/eu/2016/when-bad-screenshots-happen-to-good-writers-swapnil-ogale/>`_ from WTD EU 2016.
+
+--------------------------------------
+Report from the book club: managing up
+--------------------------------------
+
+This winter, the `Write the Docs book club <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ is reading and discussing their way through  Managing Writers by Richard Hamilton. The book covers a broad range of topics on the overall topic of managing docs projects and teams (both big and small). There’s been some fab discussion every week, so here’s a sample on the theme of showing your value and finding your place in the org.
+ 
+It seems that it’s often part of a tech writer’s job to manage up, teaching both your peers and higher-ups about the value that both you and docs bring. Not easy, but the book club has some tips!
+
+First up, behave like other, more-established roles. If they document their processes in a wiki, do the same. Announce big docs changes just as you’d announce product changes.
+
+Also, look at the interpersonal dynamics. Tech writing often needs a lot of input from other roles - if this is seen as a zero-sum exchange that doesn’t benefit the other person, it’s not gonna work. You can mitigate this by doing your research up front to show you respect their time; by seeking out opportunities to help them with other things, so you’re giving back; and by emphasising your common goals, so it’s not us vs them. See if you can make things easier for people who are helping you - a little bending the rules can be the right call sometimes.
+
```

Comment:
Last sentence doesn't seem very clearly connected here.

Reply:
You're right, Removed.